dlt/destinations/impl/redshift/redshift.py

```diff
@@ -76,15 +78,23 @@ def run(self) -> None:
         credentials = ""
```

Comment:
line 75 says it already: "we assume s3 credentials were provided"

its just not clear if they use iam-role or access+secret-key. the if-else here is a bit confusing.
i think  its safe to assume that its always AwsCredentials is always there here

Reply:
so switching from if-else to else? :)